src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -264,30 +264,83 @@ runTestCase(const std::shared_ptr<xrt_core::device>& _dev, const std::string& py
   }
   // log xclbin path for debugging purposes
   logger(_ptTest, "Xclbin", xclbinPath);
+  auto json_exists = [xclbinPath]() {
+    boost::filesystem::path test_dir(xclbinPath);
+    return boost::filesystem::exists(test_dir.parent_path().string() + "/platform.json") ? true : false;
```

Comment:
Magic string.  Please define and use the string and use boost::filesystem::path to join the paths.  For example:
```
static std::string PLATFORM_METADATA = "platform.json";
boost::filesystem::path platform_file(xclbinPath);
platform_file=/ PLATFORM_METADATA;
return boost::filesystem::exists(platform_file) ? true : false;
```

Reply:
Fixed